keycloak.js

```diff
@@ -60,6 +60,10 @@ var CheckSso = require('./middleware/check-sso');
  */
 function Keycloak (config, keycloakConfig) {
   // If keycloakConfig is null, Config() will search for `keycloak.json`.
+  if (keycloakConfig === undefined)
+  {
+    console.error(`Keycloak called with no keycloakConfig specified`);
```

Comment:
Please, do not add console output

Reply:
I was following the exiting code, which includes a bunch of console.log and console.error calls. If you want this removed I can, but I added it as I spent about 2 hours tracking down a call which ended up calling this function without a keycloakConfig and thought it may be a good idea to add this.

The better way is to rework ALL of the console calls in the code (apart from maybe the tests) to use the NPMJS debug package, which would allow devs to conditionally turn on/off debug output. But this is definitly would not be part of this PR.

Let me know if you want me to leave this in or remove it.